src/app/(auth)/login/page.tsx

```
@@ -1,13 +1,13 @@
 'use client';
 import { InputWindow } from '@/app/components/InputWindow';
-import { Login } from '@/app/components/Login';
+import { Login } from '@/app/components/logIn/Login';
 import React, { useEffect, useState } from 'react';
```

ESLint

[error] 'useEffect' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)